contracts/vault/IVault.sol

```diff
@@ -110,12 +115,19 @@ interface IVault {
      *
      * @param _to address to consider.
      *
-     * @return amount of yield for @param _to.
+     * @return amount of yield for @param _to, share of yield, and performance fee from yield
```

Comment:
hmm, 
`@return the yield, shares, and performance fee for @param _to` this does not make sense to me.
because performance fee is from yield, not `@param _to(user)`.

Reply:
ok, then something different maybe? If you don't have any ideas I can suggest something more verbose:

```
@return yield for @param _to, the performance fee for that yield, and the shares for the yield plus the performance fee.
```